lib/Doctrine/ORM/Configuration.php

```diff
@@ -150,11 +150,11 @@ public function setMetadataDriverImpl(MappingDriver $driverImpl)
      * Adds a new default annotation driver with a correctly configured annotation reader. If $useSimpleAnnotationReader
      * is true, the notation `@Entity` will work, otherwise, the notation `@ORM\Entity` will be supported.
      *
+     * @param string|string[] $paths
+     * @psalm-param string|list<string> $paths
```

Comment:
In case you are wondering about the `Invalid annotations group` message, it's about this: `@param` annotations should be grouped together, and `@psalm-param` should be at the end. `phpcbf` will fix that for you.

Reply:
Yes, this is what I also meant in my first comment. I like to keep the order of the params. Even if we want to add a another entry for `@psalm-param`.  Especially don't add it behind the return, so this was intentional.
If you think that is bad and want to keep this rule, we should follow one order.
E.g. 
```php
/**  @param
     @param

     @param-psalm
     @param-psalm

     @result
     
     @param-result

     @throws
*/
```

Or first param + result + throws and then all psalm stuff?